.github/workflows/release.yml

```diff
@@ -12,21 +12,30 @@ jobs:
         with:
           # This makes action fetch all Git history so that Changesets can generate changelogs with the correct commits
           fetch-depth: 0
+
+      - name: Cache node modules
+        id: cache-npm
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-node-modules
+        with:
+          key: reaction-v6-node-modules-${{ hashFiles('package.json') }}-${{ hashFiles('pnpm-lock.yaml') }}
+          path: '**/node_modules'
+
       - name: Use Node.js 14.x
         uses: actions/setup-node@v2
         with:
           node-version: 14
 
-      - name: Install npm 8.1.0
-        run: npm i -g npm@8.1.0
+      - name: Install pnpm
+        run: npm i -g pnpm@latest
```

Comment:
I think we had agreed to pin npm at a particular version? There has been problems with it pulling in a breaking version of npm